sdk/eventhub/eventhubs-checkpointstore-tables/README.md

```diff
@@ -0,0 +1,205 @@
+# Azure Event Hubs Checkpoint Store client library for JavaScript
+
+An Azure Blob storage based solution to store checkpoints and to aid in load balancing when using `EventHubConsumerClient` from the [@azure/event-hubs](https://www.npmjs.com/package/@azure/event-hubs) library
```

Comment:
There are several references to Blob storage. Can you remove those and update them to Azure Table Storage where appropriate? That includes links as well.